engine/table/src/main/java/io/deephaven/engine/table/impl/replay/ReplayerInterface.java

```diff
@@ -3,16 +3,16 @@
  */
 package io.deephaven.engine.table.impl.replay;
 
+import io.deephaven.base.clock.Clock;
 import io.deephaven.engine.table.Table;
-import io.deephaven.time.DateTime;
 
 import java.io.IOException;
 import java.util.TimerTask;
 
 /**
  * An interface for replaying historical data as simulated real-time data.
  */
-public interface ReplayerInterface {
+public interface ReplayerInterface extends Clock {
```

Comment:
Thoughts on making the replayer instance a `Clock` vs returning a `Clock`?  I might lean to the second, since it seems funny to say that the replayer is a clock.  Also, "is a `Clock`" is causing a bunch of methods to be duplicated in many files.

Reply:
I'm happy to change this to "replayer returns a clock" but it's not going to solve any of the duplication - the replayer and classes that extend replayer seem to be built fundamentally different, so I don't think there's an easy path to de-duplication.

Reply:
I did find a slight way to simplify this by doing "returns a Clock" - and I agree it's a better interface.